### PR TITLE
add new layer ch.swisstopo.geologie-eiszeit-lgm

### PR DESCRIPTION
**Pull request by daguer**

### CODE REVIEW

**Comment by AFoletti** [comment]
I think it would be nice to add the measurement units to the legend
Could you also please check the translations? You are removing some of them

**Comment by cedricmoullet** [comment]
@daguer please rebase

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -3628,8 +3634,9 @@ msgstr "Division General Geol. Map 200"
 msgid "ch.swisstopo.geologie-generalkarte-ggk200.metadata.autor"
 msgstr "Author(s)"
 
+#. TODO
 msgid "ch.swisstopo.geologie-generalkarte-ggk200.metadata.ggk_nr"
-msgstr "Sheet number"
+msgstr "ch.swisstopo.geologie-generalkarte-ggk200.metadata.ggk_nr"
```
Comment (cedricmoullet) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: Is it correct to remove this translation ?

Comment (AFoletti) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: no, it's not correct, but it will be solved by a rebase

Comment (cedricmoullet) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: I merge or do you want to rebase this on ?

Comment (AFoletti) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: I would like to see the branch rebased. Should not take long. @daguer ?

**Pull request merged**